health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -766,175 +766,176 @@
+             return <p>{t(Digit.Utils.locale.getTransformedLocale("MICROPLAN_STATUS_" + value))}</p>;
+           } else {
+             return (
+               <div>
+                 <p>{t("NA")}</p>
+               </div>
+             );
+           }
+ 
+         case "CAMPAIGN_DISEASE":
+           if (value && value != "NA") {
+             return <p>{t(Digit.Utils.locale.getTransformedLocale("MICROPLAN_DISEASE_" + value))}</p>;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)